[update] Install self-hosted TimescaleDB on Debian-based systems (#1761)
timescale/docs: install/installation-debian.md

    </terminal>

1.  Install Timescale GPG key

    ```bash
    wget --quiet -O - https://packagecloud.io/timescale/timescaledb/gpgkey | sudo apt-key add -
    ```

    <highlight type="note">

>     For Ubuntu 21.10 and later use the following command to install Timescale

Reviewer: ```suggestion
    For Ubuntu 21.10 and later use this command to install Timescale
```

<procedure>

### Setting up the TimescaleDB extension

   Before you create a TimescaleDB extension 

1.  Restart the service after enabling TimescaleDB with `timescaledb-tune`:

    ```bash
    systemctl restart postgresql
    ```

1.  On your local system, at the command prompt, connect to the PostgreSQL
    instance as the `postgres` superuser:

    ```bash

>     -u postgres psql

Author: <img width="733" alt="Screenshot 2022-11-14 at 10 10 50 AM" src="https://user-images.githubusercontent.com/37059749/201576729-8dccafa9-3e7f-484a-b883-eac486b50c97.png">

>     -u postgres psql

Author: yes!!! as a root user we cannot add sudo to the commands as per the guidelines.

>     -u postgres psql

Reviewer: but `-u` is a flag to the sudo command (the command is `sudo`, the flag is `-u`, and the variable is `postgres`). It's saying "change to the postgres user, and execute the psql command". So you can have `sudo -u postgres psql` or just `psql`, but not a combination of both.

>     -u postgres psql

Author: Thanks for the clarification @Loquacity !!!!

>     -u postgres psql

Author: So i leave it as is right?

>     -u postgres psql

Reviewer: possibly the better command is just tell them to run `psql` as the `postgres` user, without giving them the command to switch user.

    ```

    If your connection is successful, you'll see a message like this, followed
    by the `psql` prompt:

    ```bash
    psql (15.0 (Ubuntu 15.0-1.pgdg20.04+1), server 14.5 (Ubuntu 14.5-2.pgdg20.04+2))
    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, compression: off)
    Type "help" for help.
    ```

> 1.  Set the password for the `postgres` user using:

Reviewer: ```suggestion
1.  Set the password for the `postgres` user:
```

    ```sql
    \password postgres 
    ```

> 1.  Exit from PostgreSQL using the command `\q`.

Reviewer: Can't do it as a suggestion because GH markup, but put this command in a block instead of inline.

Author: done.
